src/components/AddItems.js

```diff
@@ -19,38 +26,44 @@ const AddItemsToList = () => {
   function submitShoppingListItemHandler(event) {
     event.preventDefault();
 
-    const userToken = localStorage.getItem('token');
-
     if (shoppingListItemName === '') {
       alert('Please enter item name...');
       return;
     }
 
-    const values = {
+    const item = {
       shoppingListItemName,
       daysLeftForNextPurchase,
       lastPurchasedOn: null,
     };
 
-    db.where('token', '==', userToken)
-      .get()
-      .then((data) => {
-        if (data.docs.length) {
-          db.doc(data.docs[0].id).update({
-            items: arrayUnion(values),
-          });
-          setShoppingListItemName('');
-          setDaysLeftForNextPurchase(7);
-        } else {
-          db.add({
-            token: userToken,
-            items: [values],
-          });
-          setShoppingListItemName('');
-          setDaysLeftForNextPurchase(7);
-        }
+    if (shoppingList.length) {
+      const { documentId } = shoppingList[0];
+
```

Comment:
I love the use of react-firebase-hooks, this is great!! I guess we just need to be careful when Joseph and I merge our code that our changes to AddItems are inserted in the right place, now that this section has changed!

Reply:
yea exactly